src/main/edit_delete/postProcess.ts

```diff
@@ -0,0 +1,66 @@
+import { Transcription } from 'sharedTypes';
+import Clip from './Clip';
+
+/**
+ * Removes deleted words from edited trancription
+ * @param wordList
+ */
+const removeDeleted = (wordList: Array<any>): void => {
+  let i = 0;
+  while (i < wordList.length) {
+    if (wordList[i].deleted) {
+      wordList.splice(i, 1);
+    } else {
+      i += 1;
+    }
+  }
+};
+
+/**
+ * Processes edited data to be ready for export
+ * @param wordList
+ * @returns
+ */
+const FILENAME = 'placeholder_filename.mp4';
+
+const postProcess = (jsonTranscript: Transcription): Array<Clip> => {
+  const wordList = jsonTranscript.words;
+
+  removeDeleted(wordList);
```

Comment:
if you do it immutably you will then need to reassign wordList - probably easies to just do it in the initial assignment

`const wordList = removeDeleted(jsonTranscript.words)`

Reply:
Makes sense, thanks. I've done it as
 "const wordList = jsonTranscript.words.filter(word => !deleted)".

Am I correct in saying that this wouldn't work:
const wordList = jsonTranscript.words
const wordList = wordList.filter(word => !deleted)
?

Reply:
That would throw a syntax error because you'd be reassigning a const